doc/api/v8.md

```diff
@@ -61,12 +61,21 @@ following properties:
 }
 ```
 
-## `v8.getHeapSnapshot()`
+## `v8.getHeapSnapshot([options])`
 
 <!-- YAML
 added: v11.13.0
+changes:
+  - version: REPLACEME
+    pr-url: REPLACEME
+    description: Support options to configure the heap snapshot.
 -->
 
+* `options` {Object}
+  * `exposeInternals` {boolean} If true, expose internals in the heap snapshot.
```

Comment:
I am actually not sure if we should use constants (e.g. strings) for these, suggestions welcome. Also AFAIK `exposeInternals` doesn't currently change anything for us because we don't use any embedder tracing mechanism provided by V8 that could would make a difference here, but this might change in the future.